### PR TITLE
Deploying Keras Models To Android

### LINKED ISSUE #11778: Create tutorial on how to deploy Keras models to Android.
Everything is in the title.

Note created by @fchollet in the "Requests for contributions".

### PR DESCRIPTION
**Pull request by saadmanrafat**
### Summary
Tutorial on how to deploy Keras model to Android

### Related Issues
Fixes #11778 

### PR Overview

- [n] This PR requires new unit tests [y/n] (make sure tests are included)
- [y] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [y] This PR is backwards compatible [y/n]
- [n] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

File: docs/templates/deploy-to-android.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+Refer to this [link](https://www.tensorflow.org/lite/convert/cmdline_reference) for the complete command line reference.
+
+## Out of the Box Models
+Here's a list of [out of the box models](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/models.md) which can be readily deployed in mobile devices.
+
+## Embedding The Model
+
+TensorFlow provides a [step-by-step walkthrough and an Android app skeleton](https://codelabs.developers.google.com/codelabs/tensorflow-for-poets-2-tflite/#3) to just plug and play (or even deploy) with the model, without having any first hand experience of Android Development.
```
Comment (fchollet) [inline docs/templates/deploy-to-android.md]: I feel like the current tutorial should provide an end-to-end example (while also linking to further resources like this one). In its current form this tutorial seems too barebones. It only shows how to use the TFLite converter.

**Comment by fchollet** [comment]
Closing PR. Please reopen a PR on the Keras blog repo, with a more fleshed out version of this tutorial. Thank you!

**Comment by saadmanrafat** [comment]
Thanks for the feedback I will try to make the aforementioned changes also about the examples on deploying, should I add TFLite Java API Interpreter examples?

**Comment by skonto** [comment]
@saadmanrafat if you need some inspiration for an end to end example I have done this here: https://github.com/skonto/talks/blob/master/spark-summit-2018-London/models/iris.py
and here: 
https://github.com/skonto/talks/blob/master/spark-summit-2018-London/mltest/feature/src/main/java/com/github/com/ml_test/feature/IrisClassifier.java

**Comment by saadmanrafat** [comment]
@skonto thanks. I will take a look at them.

**Pull request closed without merging**